src/framework/audio/internal/audiobuffer.cpp

```diff
@@ -81,15 +76,15 @@ void AudioBuffer::pop(float* dest, unsigned int sampleCount)
 
     unsigned int from = m_readIndex;
     auto memStep = sizeof(float);
-    auto to = m_readIndex + sampleCount * m_streamsPerSample;
+    auto to = m_readIndex + sampleCount * config()->requiredAudioChannelsCount();
```

Comment:
These are methods call very often, maybe better to write AudioChannelsCount to member and use it

Reply:
added a member m_audioChannelsCount